sample/scripts/initSampleCode.ts

```diff
@@ -0,0 +1,44 @@
+export default function initSampleCode() {
+    insertHtmlSampleCode();
+    insertJsSampleCode();
+
+    var fileref=document.createElement('script');
+    fileref.setAttribute("type","text/javascript");
+    fileref.setAttribute("src", "https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js");
+    document.getElementsByTagName("head")[0].appendChild(fileref);
+}
+
+function insertHtmlSampleCode() {
+    let htmlSampleCode = document.getElementsByClassName('htmlSample')[0];
+    let codeString: string = [
+        '<html>',
+        '   <body>',
+        '       <div id="editorDiv" style="width: 500px; height: 300px; overflow: auto;',
+        '       border: solid 1px black"></div>',
+        '       <script src="editor.js"></script>',
+        '   </body>',
+        '</html>',
+    ].join("\n");
+    htmlSampleCode.appendChild(createElementFromContent('htmlSampleCode', codeString, 'prettyprint lang-html'));
+}
+
+function insertJsSampleCode() {
+    let jsSampleCode = document.getElementsByClassName('jsSample')[0];
+    let codeString: string = [
+        'var roosterjs = require(\'roosterjs\');',
+        'var editorDiv = document.getElementById(\'editorDiv\');',
```

Comment:
Since we have limited possibilities of code, I think we can just hard-code the styles and no need to format them in runtime.

Reply:
I found it harder than thought to hard code the styles. Basically we need to wrap each string segment with a span to set color. This makes the code hard to read. Currently I will just remove the dependency on code-prettify.